### ISSUE TITLE
Accept Not working on file drop

### ISSUE THREAD

**Issue by shashank100** [paramaggarwal/react-dropzone#57]

**Comment by okonet** [comment]
+1

**Comment by paramaggarwal** [comment]
I'm not sure what this property means. Could you submit a PR?

**Comment by paramaggarwal** [comment]
Yes, the `accept` prop was misleading. `v2.0.1` removes support for it.

**Comment by okonet** [comment]
I'm not sure that removing support for `accept` is a good idea... It is in the HTML5 standard https://developer.mozilla.org/en/docs/Web/HTML/Element/Input and it would be cool to have support for it.

**Comment by paramaggarwal** [comment]
Yes, it works on `input` but no clear way to have it work on a dropped file. Will accept PRs if someone can figure it out.

**Comment by okonet** [comment]
Ok, I'll do that. Have a solution in place.

**Comment by iansinnott** [comment]
@okonet Can you  confirm that the pull request works in `2.1.0`? I get an error coming from the `attr-accept` module when I drag a file onto the dropzone:

```
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
```

Specifically the error is generated on line 21: https://github.com/okonet/attr-accept/blob/master/src/index.js#L21

The issue seems to be that the variable `file` is not an instance of `File` but instead is an instance of `DataTransferItem`, which has no `name` property.

**Comment by okonet** [comment]
I could not find any release notes to 2.1.0. What's changed?

**Comment by iansinnott** [comment]
What version works for you? Given the error that was getting thrown in my console it seems that it's a matter of being able to handle the `file` var bing an instance of `DataTransferItem` as opposed to `File`. I'm also not sure if this changed in a recent version of react-dropzone.

**Comment by okonet** [comment]
Interestingly I get `TypeError: can't convert undefined to object` in Firefox after updating to 2.1.0

What browser are you using?

**Comment by okonet** [comment]
But this one got fixed here: https://github.com/paramaggarwal/react-dropzone/commit/df6ed1290d007b7bf16fff81f79dfef0e0776046

The thing is that it never get released... I think you should publish a new version to npm.

**Comment by iansinnott** [comment]
If it helps, I've been using the lib on Chrome 48 (Canary).

**Comment by okonet** [comment]
I've just checked 2.1.0 in Chrome 48 (Canary) and everything works as expected. Can you provide me with better steps to reproduce? What file did you use and what settings? Mind adding an example to github please?

**Comment by iansinnott** [comment]
Of course. I'll see if I can repro in a fresh git repo with the bare minimum of dependencies. Thanks for the help @okonet

**Comment by paramaggarwal** [comment]
Published `v2.2.0` to npm from master.
